test/functional/global-options.spec.ts

```diff
@@ -84,13 +89,18 @@ describe("routing-controllers global options", () => {
 
     describe("when routePrefix is used all controller routes should be appended by it", () => {
     
-        let expressApp: any, koaApp: any;
-        before(done => expressApp = createExpressServer({ routePrefix: "/api" }).listen(3001, done));
-        after(done => expressApp.close(done));
-        before(done => koaApp = createKoaServer({ routePrefix: "/api" }).listen(3002, done));
-        after(done => koaApp.close(done));
+        let apps: any[] = [];
+        before(done => apps.push(createExpressServer({ routePrefix: "/api" }).listen(3001, done)));
+        before(done => apps.push(createExpressServer({ routePrefix: "api" }).listen(3002, done)));
+        before(done => apps.push(createKoaServer({ routePrefix: "/api" }).listen(3003, done)));
+        before(done => apps.push(createKoaServer({ routePrefix: "api" }).listen(3004, done)));
+        after(done => { apps.forEach(app => app.close()); done(); });
     
-        assertRequest([3001, 3002], "post", "api/users", { firstName: "Umed", lastName: "Khudoiberdiev" }, response => {
+        assertRequest([3001, 3002, 3003, 3004], "post", "api/users", { firstName: "Umed", lastName: "Khudoiberdiev" }, response => {
+            expect(response).to.have.status(200);
```

Comment:
@felipesabino any progres? I would like to approve this PR but you have to fix the tests 😉